amazon-kinesis-client/src/main/java/software/amazon/kinesis/lifecycle/ConsumerStates.java

```diff
@@ -146,7 +148,7 @@ public ConsumerState successTransition() {
 
         @Override
         public ConsumerState shutdownTransition(ShutdownReason shutdownReason) {
-            return ShardConsumerState.SHUTDOWN_COMPLETE.consumerState();
+            return ShardConsumerState.SHUTTING_DOWN.consumerState();
```

Comment:
Ended not creating a new state as earlier discussed, I figured out that what was going on with the state transitions was that BlockedOnParent state was transitioning to ShutdownComplete which circumvents shutdown task, I don't think there's any concern here with always doing shutdown task for the transition here, but if you feel strongly otherwise we can use the intermediary state as earlier discussed.